Dockerfile

```diff
@@ -1,18 +1,26 @@
-FROM golang:1.15 as bd
-WORKDIR /github.com/layer5io/meshery-traefik-mesh
-ADD . .
-RUN GOPROXY=https://proxy.golang.org GOSUMDB=off go build -ldflags="-w -s" -a -o /meshery-traefik-mesh .
-RUN find . -name "*.go" -type f -delete
+FROM golang:1.15 as builder
 
-FROM alpine
-RUN addgroup -S appgroup && adduser -S appuser -G appgroup
-RUN apk --update add ca-certificates
-RUN mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
+ARG ENVIRONMENT="development"
```

Comment:
This is unfortunate (but topic for another discussion, issue and PR) because it means that the primary artifact needs to be rebuilt for each environment. This is not recommended.

Reply:
@mgfeller, although it is being passed on to the go binary using the ldflags, it is not actually being used there. I kept it for consistency with the istio dockerfile. However, I don't think the `environment` is being used in the istio adapter either. 

Would you suggest removing it from here?

Reply:
@utkarsh-pro yes, I think that would be a Good Thing :+1: